cmd/exec/main.go

```diff
@@ -49,12 +48,10 @@ func main() {
 	rootCmd.AddCommand(execCmd)
 	rootCmd.AddCommand(pushCmd)
 	rootCmd.AddCommand(setCmd)
-	rootCmd.AddCommand(eventCmd)
-	rootCmd.AddCommand(eventfilterCmd)
-	eventfilterCmd.AddCommand(createCmd)
-	eventfilterCmd.AddCommand(deleteCmd)
-	eventfilterCmd.AddCommand(updateCmd)
-	eventfilterCmd.AddCommand(applyCmd)
+	rootCmd.AddCommand(eventsCmd)
+	eventsCmd.AddCommand(sendEventCmd)
```

Comment:
Merged the apply command and the other command that sends an event. You can use the optional flag `--endpoint` to decide if it hits the default `/broadcast` vs `/broadcast/myfilter` (with `--endpoint=myfilter`), for example.